src/libp2p_ipc/dune

```diff
@@ -1,7 +1,7 @@
 (library
  (name libp2p_ipc)
  (public_name libp2p_ipc)
- (flags -w -53)
+ (flags -w -53-55)
```

Comment:
why is this needed?

Reply:
Fair question! Nix fails while building `libnp2p_ipcs`, ignoring warning `55` while building `src/libp2p_ipcs/libp2p_ipcs_capnp.ml` seems to fix it. [See this internal discussion](https://o1-labs.slack.com/archives/C03DZ0MCCFR/p1696352880596909) for more details.